<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>735</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>711</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_point_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_point_box">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="add_point_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить точку</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="delay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Задержка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>600</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрасить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>570</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Замкнуть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>630</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет линии</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_line_black">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Чёрный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_line_red">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>620</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Красный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_line_blue">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>590</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Синий</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_bg_white">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Белый</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_bg_blue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>590</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Синий</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_line_green">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Зеленый</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_line_white">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>680</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Белый</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_bg_black">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>680</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Чёрный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_bg_red">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>620</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Красный</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет фона</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_bg_green">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Зеленый</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Точки</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="points_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Время работа алгоритма в секундах</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="timing_text">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>560</y>
      <width>251</width>
      <height>85</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
